Header header
string[] joint_names
trajectory_msgs/JointTrajectoryPoint[] points
geometry_msgs/Pose end_pose